ui/desktop/app/styles/app.scss

```diff
@@ -512,12 +512,16 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
   }
 }
 
-.details-sidebar-target-details {
+.details-sidebar {
   background-color: var(--token-color-palette-neutral-50);
   border-radius: 0.313rem;
   padding: 1.5rem 2.5rem;
   margin-top: 3.5rem;
 
+  .host-detail {
```

Comment:
This works for this use-case but if we ever add another section to the sidebar we'd have to refactor this again. I feel like using  `:not(:first-child)` or `:not(:last-child)` would work better for this scenario, what do you think?

Reply:
I was wondering if we should keep it more dynamic because I believe we will add "Active Sessions" to the sidebar for the `targets/target_id` route as well. Let me play around with this.

Reply:
Updated. Let me know what you think.

Reply:
Yup I think this looks good. One last question is do we plan on using the `detail-group` class for anything else?

If not should we just do something like this since it's only used for this margin right now:
```
  > :not(:last-child) {
    margin-bottom: 2.25rem;
  }
```

Reply:
Nope, it really is just needed for spacing. This is nice. I will update to this.